packages/contracts/src/contracts/current/protocol/Exchange/MixinSettlementProxy.sol

```diff
@@ -20,22 +20,24 @@ pragma solidity ^0.4.21;
 pragma experimental ABIEncoderV2;
 
 import "./mixins/MSettlement.sol";
-import "../TokenTransferProxy/ITokenTransferProxy.sol";
 import "../../tokens/Token/IToken.sol";
 import "./LibPartialAmount.sol";
+import "../AssetProxyDispatcher/IAssetProxy.sol";
 
 /// @dev Provides MixinSettlement
 contract MixinSettlementProxy is
     MSettlement,
     LibPartialAmount
 {
-
-    ITokenTransferProxy TRANSFER_PROXY;
+    IAssetProxy TRANSFER_PROXY;
```

Comment:
Is this always an`IAssetProxyDispathcher`, or are there times when this is the lower level `IAssetProxy`? If it's always the `IAssetProxyDispathcher` I would declare the type as so and rename to `PROXY_DISPATCHER`.